src/vellum/workflows/nodes/displayable/inline_prompt_node/tests/test_node.py

```diff
@@ -81,6 +82,53 @@ def generate_prompt_events(*args: Any, **kwargs: Any) -> Iterator[ExecutePromptE
     assert len(mock_api.call_args.kwargs["input_variables"]) == 5
 
 
+def test_inline_prompt_node__coalesce_expression_serialization(vellum_adhoc_prompt_client):
```

Comment:
this is a serialization bug, we should move this to inline_prompt_node serialization tests and call serialize() instead of run()